comms/rabbitmq.go

```diff
@@ -683,19 +683,19 @@ func (rmq rabbitMQComms) ListenForGroupTasks(groupName string, dereg chan bool)
 		for d := range msgs {
 
 			// Unmarshal into BaseTaskMessage to determine type
-			var base_msg tasks.BaseTask
-			err = json.Unmarshal(d.Body, &base_msg)
+			var baseMsg tasks.BaseTask
+			err = json.Unmarshal(d.Body, &baseMsg)
 			// TODO(mierdin): Need to handle this error
 
 			log.Debugf("Agent task received: %s", d.Body)
 
 			// call agent task method based on type
-			switch base_msg.Type {
+			switch baseMsg.Type { // TODO: remove switch?
 
 			// This has been removed, as I am moving away from using queues that use the group name.
```

Comment:
I didn't look very closely at the contents of this method before. Is it unused? My .02 would be to remove it entirely and put it back if needed. Dead code, commented out code, etc should always be pruned, IMO.

If you don't want to go that route, what should the new comment say? `May want to reuse this mechanism in the future`?